### ISSUE TITLE
Build fails for Sodaq Explorer in pio, not in Arduino IDE

### ISSUE THREAD

**Issue by SRGDamia1** [platformio/platformio-pkg-framework-arduinosam#2]
I'm having builds fail for the Sodaq explorer that are succeeding when I use the Arduino IDE.  These are the errors I'm getting:  https://travis-ci.org/EnviroDIY/Arduino-SDI-12/jobs/245990633

**Comment by ivankravets** [comment]
@keestux does it link with a recent https://github.com/platformio/platformio-pkg-framework-arduinosam/pull/1 ?

**Comment by keestux** [comment]
Probably (most likely :) due to https://github.com/platformio/platform-atmelsam/pull/16

**Comment by ivankravets** [comment]
@keestux could you point me where did you find v1.6.14 core for Sodaq? I see that the latest is 1.6.10 

* http://downloads.sodaq.net/package_sodaq_index.json

**Comment by SRGDamia1** [comment]
The SodaqCore-samd repo (https://github.com/SodaqMoja/SodaqCore-samd) has branches up to 1.6.14, but only releases up to 1.6.12.1.

I tried copying the contents of 1.6.10 from the SodaqCore-samd repo over top of \.platformio\packages\framework-arduinosam\cores\sodaq, but it still wouldn't compile for me.  Maybe that's because I don't know what changes to make to the atmelsam platform to go along with it.

**Comment by ivankravets** [comment]
@SRGDamia1 I'm working on a new release of `atmelsam` development platform. I'll inform you here.

**Comment by ivankravets** [comment]
Fixed in https://github.com/platformio/platform-atmelsam/releases/tag/v2.2.0

Please `pio update` and restart Travis.CI job

**Comment by keestux** [comment]
On 26-06-17 22:46, Ivan Kravets wrote:
>
> @keestux <https://github.com/keestux> could you point me where did you 
> find v1.6.14 core for Sodaq? I see that the latest is 1.6.10
>
>   * http://downloads.sodaq.net/package_sodaq_index.json
>

Nowadays, we have split the AVR and SAMD board files. You need this URL

     http://downloads.sodaq.net/package_sodaq_samd_index.json

**Comment by ivankravets** [comment]
> we

Sorry, do you represent Sodaq? If yes, would be thankful for the further PR. Now, just only need to make PR to this repo. I think no need to modify boards manifest.

**Comment by keestux** [comment]
On 26-06-17 23:17, Sara Damiano wrote:
>
> The SodaqCore-samd repo (https://github.com/SodaqMoja/SodaqCore-samd) 
> has branches up to 1.6.14, but only releases up to 1.6.12.1.
>

Good catch. I've created the tags and pushed them to github

> I tried copying the contents of 1.6.10 from the SodaqCore-samd repo 
> over top of .platformio\packages\framework-arduinosam\cores\sodaq, but 
> it still wouldn't compile for me. Maybe that's because I don't know 
> what changes to make to the atmelsam platform to go along with it.
>

Let's try to work towards 1.6.14.

A little background. The SodaqCore-samd is a fork of ArduinoCore-samd. 
This latter is
at 1.6.14 and with SodaqCore-samd we will try to follow that versioning 
scheme with the
prefix "v" and the suffix "-sodaq". Main changes are the additional 
variants for our boards.
But we have a few extra changes to suit our needs. (E.g. we set ports 
default to output to
lower the power consumption.)

In other words, you are better off trying to work with v1.6.14-sodaq.

Notice that you need an update of platform-atmelsam as well.

**Comment by keestux** [comment]
On 27-06-17 19:26, Ivan Kravets wrote:
>
>     we
>
> Sorry, do you represent Sodaq? If yes, would be thankful for the 
> further PR. Now, just only need to make PR to this repo. I think no 
> need to modify boards manifest.
>

Yes, I'm one of the founders of SODAQ and I'm the maintainer of 
SodaqCore-samd (and a lot more of
the repos at https://github.com/SodaqMoja)

**Comment by ivankravets** [comment]
Cool! Now all should work very well!

I'll ping you if someone contact with any issue related to Sodaq.

Again, thanks a lot for your PRs!

**Comment by SRGDamia1** [comment]
Sorry, I didn't confirm, but this did fix everything.

**Comment by keestux** [comment]
Thanks for letting us know
